rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -848,6 +849,28 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 	SubscribeTooltip(ui->gb_DiskCacheClearing, tooltips.settings.limit_cache_size);
 	connect(ui->enableCacheClearing, &QCheckBox::stateChanged, ui->maximumCacheSize, &QSlider::setEnabled);
 
+	// Date Time Edit Box
+	m_emu_settings->EnhanceDateTimeEdit(ui->console_time_edit, emu_settings_type::ConsoleTimeOffset, tr("dd/MMM/yyyy HH:mm", "Console time offset"), true, true);
+	connect(ui->console_time_reset, &QAbstractButton::clicked, [this]()
+	{
+		ui->console_time_edit->setDateTime(QDateTime::currentDateTime());
+	});
+	// We set the minimum/maximum dates on the control to be +/- 99years (this is one year shorter than the second limits, just to ensure we never hit the hard limits)
+	ui->console_time_edit->setDateTimeRange(QDateTime::currentDateTime().addYears(-99), QDateTime::currentDateTime().addYears(99));
+	ui->console_time_edit->setMaximumDateTime(QDateTime::currentDateTime().addYears(99));
```

Comment:
don't do these here. do these in the Enhance function and use the min and max provided by the setting